content/concepts/multiplex/yamux.md

```diff
@@ -1,15 +1,34 @@
 ---
-title: "yamux"
+title: "Yamux"
 description: "yamux is a multiplexing protocol designed by Hashicorp."
 weight: 180
 ---
 
-[yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed by [Hashicorp](https://www.hashicorp.com/).
-yamux offers more sophisticated flow control than mplex, and can scale to thousands of multiplexed streams over a
-single connection.
+## What is Yamux?
+
+[Yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed
+by [Hashicorp](https://www.hashicorp.com/) for Go, built over TCP.
+Yamux provides a high-level interface for working with SCTP streams. It is inspired
+by [SPDY](https://en.wikipedia.org/wiki/SPDY), but it is not interoperable with it.
+
+Yamux offers more sophisticated flow control than [mplex](mplex) while also supporting
+backpressure, which is a mechanism that helps to prevent data from being sent faster
+then it can be processed.
+
+Backpressure allows the receiver of a data stream to specify an offset
+to which the sender can send data. This offset increases as the receiver process the data,
+allowing the sender to send more data. This helps prevent the
+sender from sending more data than the receiver can handle, leading to buffering
+and potentially causing the application to crash. This is especially important in
+applications where the receiver might be slower at processing the data, such as when the
+receiver is a low-powered device or when the data is complex and requires a lot of processing.
 
 {{< alert icon="💡" context="note">}}
-This section is incomplete, and many of the articles are stubs. To help fill in
-the gaps, please see the issues linked in each article to add your input and
-help us prioritize the outstanding work.
+**Yamux should be used over mplex in libp2p**. It is well-suited for applications that require fast,
+reliable data transfer, such as real-time communication or streaming media as it is optimized for low-latency, high-bandwidth environments.
```

Comment:
Not really. When you're streaming media, you typically want an unreliable transport.